src/AgGrid.elm

```diff
@@ -304,6 +304,8 @@ type alias ColumnDef dataType =
 -}
 type alias ColumnSettings =
     { aggFunc : Aggregation
+    , allowedAggFuncs : List Aggregation
```

Comment:
According to the docs: 
> Aggregation functions allowed on this column e.g. ['sum', 'avg']. If missing, all installed functions are allowed. 

An empty list is not sufficient for using the default aggregations. We probably need to set it to `null` for that. An empty list means that no aggregations are allowed.

So, we probably want to make this a `Maybe (List Aggregation)`

Reply:
Done - I guess I misinterpreted "is missing" here.